index.js

```diff
@@ -309,14 +309,23 @@ function runDataIntegrityProofFormatTests({
  * @param {string} [options.testDescription] - An option to define
  *   the test description. The default value is set to
  *   `Data Integrity (verifier)`.
+ * @param {object} options.testDataOptions - Options for test data creation
+ *   such as suite.
+ * @param {object} [options.optionalTests] - Options for running
+ * optional tests from DataIntegrity such as created and authentication.
  *
- * @returns {object} Returns the test suite being run.
+ * @returns {describe} Returns the test suite being run.
```

Comment:
actually I tested this and describe is not a promise which is kind of annoying, there are definitely occasions where you might want to `await describe` so that one suite finishes before another runs. The mocha source code suggests that `describe` is actually a `Suite` https://github.com/mochajs/mocha/blob/2f3fedcc41cbb9d3e503d84098fcc07d7c3c49f1/lib/mocha.js#L110-L112

https://github.com/mochajs/mocha/blob/2f3fedcc41cbb9d3e503d84098fcc07d7c3c49f1/lib/suite.js

you can also verify that describe returns a suite by doing

```js
const d = describe('foo', function() {
  return 'bar'
})
console.log(d)
// instance of a Suite here
```

I actually always though describe returned a promise to, but it returns a Suite.

Reply:
So `@return` should be `Suite`, correct?

Reply:
@BigBlueHat I changed it back to `object` to match the other `@returns` with a note that the object is a suite: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/pull/37/commits/d782f1f60077b9d7a19c4c90baac2ea94ee448a2